shared/runtime/tinyusb_helpers.c

```diff
@@ -52,3 +52,32 @@ void tud_cdc_line_state_cb(uint8_t itf, bool dtr, bool rts) {
 }
 
 #endif
+
+
+#if MICROPY_HW_USB_MSC_EXCLUSIVE_ACCESS
+#include "tusb.h"
+#include "extmod/vfs.h"
+#include "extmod/vfs_fat.h"
+static mp_sched_node_t mp_remount_sched_node;
+
+STATIC void tud_msc_remount_task(mp_sched_node_t *node) {
+    mp_vfs_mount_t *vfs = NULL;
+    for (vfs = MP_STATE_VM(vfs_mount_table); vfs != NULL; vfs = vfs->next) {
+        if (vfs->len == 1) {
+            const char *path_str = "/";
+            mp_obj_t path = mp_obj_new_str(path_str, strlen(path_str));
```

Comment:
The mount point may be different for different ports. mimxrt uses /flash as default mount point, to make a difference to /sdcard. But that's a minor issue. Could be solved by using a #define.

Reply:
That's true, this code could use some helper from vfs to get the vfs used for MSC (and maybe a helper to remount as read/write).

Reply:
Could be more simple: The mp_vfs_mount_t structure contains the mount point. So it could simply be re-used.
```
typedef struct _mp_vfs_mount_t {
    const char *str; // mount point with leading /
    size_t len;
    mp_obj_t obj;
    struct _mp_vfs_mount_t *next;
} mp_vfs_mount_t;
```